server/ast/expr.go

```diff
@@ -464,7 +464,15 @@ func nodeExpr(node tree.Expr) (vitess.Expr, error) {
 	case *tree.Subquery:
 		return nodeSubquery(node)
 	case *tree.Tuple:
-		return nil, fmt.Errorf("tuples are not yet supported")
+		valTuple := make(vitess.ValTuple, len(node.Exprs))
```

Comment:
We need to do something with the `Labels` field. This goes for all expressions and nodes in general, all fields need to be accounted for in some way. Generally, I'll return an error if something is used that we don't yet have any support for. I've not looked into exactly how the labels are supposed to be used, but if we don't support the functionality yet, then we need to throw an error. If the labels are being used and we need to get past the error (such as sqllogictests are failing), then add an explicit TODO regarding the labels. We need something here though, otherwise it looks like we've fully "completed" the `*tree.Tuple` type, when it needs more work.

The same applies to the `Row` field too. The `typ` field can be ignored, it's a leftover from the execution engine that's been _mostly_ removed.

Anywhere that you see I'm not taking into account all of the fields is a mistake, and it should be corrected. I like returning errors because it gives people something that they can file issues for, and also helps to enforce that it works when we _know_ it works, which I also think will help with gaining outside confidence in the tool.